python/ray/llm/_internal/serve/deployments/llm/vllm/vllm_models.py

```diff
@@ -147,10 +147,13 @@ def placement_strategy(self) -> str:
 
     @property
     def placement_bundles(self) -> List[Dict[str, float]]:
-        bundles = [
-            {"GPU": 1, self.ray_accelerator_type(): 0.001}
-            for _ in range(self.num_gpu_workers)
-        ]
+        if self.accelerator_type is None:
+            bundles = [{"GPU": 1} for _ in range(self.num_gpu_workers)]
+        else:
+            bundles = [
+                {"GPU": 1, self.ray_accelerator_type(): 0.001}
+                for _ in range(self.num_gpu_workers)
+            ]
```

Comment:
Maybe do something like this to be more clean
```suggestion
        bundle = {"GPU": 1}
        if self.accelerator_type:
            bundle[self.ray_accelerator_type()] = 0.001
        bundles = [bundle for _ in range(self.num_gpu_workers) ]
```